### PR TITLE
Addded Date documentation

### PR DESCRIPTION
**Pull request by Bcrunk**

### CODE REVIEW

File: packages/storybook-html/src/components/core/date.story.mdx [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+          <td>
+          10:00 AM <br />
+          2:30 PM
+          </td>
+          <td><code>hh:mm AM/PM</code></td>
+          <td>For time, use uppercase AM or PM without periods, separated from time with a space. <br />
```
Comment (uqnm) [inline packages/storybook-html/src/components/core/date.story.mdx:35]: need to probably confirm this whether upper or lower case

File: packages/storybook-html/src/components/core/date.story.mdx [hunk 2]
```diff
@@ -0,0 +1,285 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+
+<Meta title="Core/Standards/Date formats" />
+
+# Date Formats
+
+## Overview
+
+Display data such as date & time in a consistent and user-friendly way. Present the information concisely and avoid unnecessary redundancy.
```
Comment (uqnm) [inline packages/storybook-html/src/components/core/date.story.mdx:9]: think the word unnecessary is...... unnecessary (see what i did there?)

File: packages/storybook-html/src/components/core/date.story.mdx [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+        <tr>
+          <td>
+          0:30 <br />
+          2:14 <br />
+          1:01:05 <br />
+          </td>
+          <td><code>hh:mm:ss</code></td>
+          <td>
+          Use HH:MM:SS to show the duration of a recording, like audio or video. If hours or seconds don’t apply, then omit them.
+          There is no need to specify hour, minute, second units (ie. 1hr 1min 5 sec)
```
Comment (uqnm) [inline packages/storybook-html/src/components/core/date.story.mdx:207]: space between 5 and sec

**Review by uqnm — COMMENTED** [review]
added a few comments - let me know what you think

File: packages/storybook-html/src/components/core/date.story.mdx (hunk 1 above)
Comment (Bcrunk) [reply packages/storybook-html/src/components/core/date.story.mdx:35]: I will add a ticket to review this with the content team as we have conflicting advice.

**Review by uqnm — APPROVED** [review]
approved

**Pull request merged**